### PR TITLE
[Scopes] sort properties with numbers in ascending order

### PR DESCRIPTION
**Pull request by irfanhudda**
Associated Issue: #1991

### Summary of Changes

* Add function to `ObjectInspector.js` to sort (numbers in ascending order and strings in lexicographical order.)

### Screenshots/Videos (OPTIONAL)
object

### CODE REVIEW

File: src/components/shared/ObjectInspector.js [hunk 1]
```diff
@@ -61,10 +61,21 @@ function isDefault(item) {
   return WINDOW_PROPERTIES.includes(item.name);
 }
 
+function sortProperties(properties) {
+  return properties.sort((a, b) => {
+    // Sort numbers in ascending order and sort strings lexicographically
+    const aInt = parseInt(a, 10), bInt = parseInt(b, 10);
```
Comment (jasonLaster) [inline src/components/shared/ObjectInspector.js]: nit:

```js
const aInt = parseInt(a, 10);
const  bInt = parseInt(b, 10)
```

File: src/components/shared/ObjectInspector.js [hunk 2]
```diff
@@ -61,10 +61,21 @@ function isDefault(item) {
...
+    if (isNaN(aInt) || isNaN(bInt)) {
+      return a > b ? 1 : -1;
+    }
+    return aInt - bInt;
```
Comment (jasonLaster) [inline src/components/shared/ObjectInspector.js]: newline

**Comment by jasonLaster** [comment]
Could you attach screenshots with an object with string keys and a mixed example?

**Comment by irfanhudda** [comment]
@jasonLaster Here's an example with both types of keys
mixed_obj

**Pull request merged**